### PR TITLE
Updated README [Work In Progress]

### PR DESCRIPTION
**Pull request by amad-person**
Things done:
- short introduction added
- main features section added

Things that still need to be done:
- UI mockup
- marketing blurb

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -7,6 +7,10 @@ https://coveralls.io/github/CS2103JAN2018-W13-B1/main?branch=master[image:https:
 https://www.codacy.com/app/CS2103-W13-B1-helper/main?utm_source=github.com&utm_medium=referral&utm_content=CS2103JAN2018-W13-B1/main&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/bd21fc56cd1f4c968e246a55d23ed930[Codacy Badge]]
 https://gitter.im/se-edu/Lobby[image:https://badges.gitter.im/se-edu/Lobby.svg[Gitter chat]]
 
+ContactSails is an address book application meant exclusively for salespersons from small businesses with customers from various social media platforms.
+
+In addition to aggregating customer details from these platforms, it can also be used for keeping track of orders and sending relevant promotions to customers.
```
Comment (SuxianAlicia) [inline README.adoc]: Maybe we can phrase it as "keeping track of orders from customers and sending relevant promotions to them."?

File: README.adoc [hunk 2]
```diff
@@ -15,24 +19,26 @@ ifndef::env-github[]
...
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+== Features
+
+With ContactSails, you can do the following:
+
```
Comment (SuxianAlicia) [inline README.adoc]: Perhaps add the sentence "Manage customer contacts: Add, edit and delete customer contact details." ?

**Comment by SuxianAlicia** [comment]
Added some comments and phrasing and such. Looks good and ready to be merged.

File: README.adoc (hunk 1 above)
Comment (amad-person) [reply README.adoc]: Noted, will update

File: README.adoc (hunk 2 above)
Comment (amad-person) [reply README.adoc]: Adding, thanks!

**Pull request merged**